modules/inflowwind/src/InflowWind.f90

```diff
@@ -240,6 +240,20 @@ SUBROUTINE InflowWind_Init( InitInp,   InputGuess,    p, ContStates, DiscStates,
       END IF
 
          ! initialize sensor data:   
+         p%lidar%NumBeam = InputFileData%NumBeam
+         p%lidar%RotorApexOffsetPos = InputFileData%RotorApexOffsetPos
+         p%lidar%SensorType = InputFileData%SensorType      
+         p%lidar%LidRadialVel   = InputFileData%LidRadialVel
+         p%lidar%NumPulseGate = InputFileData%NumPulseGate
+         p%lidar%FocalDistanceX =  InputFileData%FocalDistanceX
```

Comment:
Were the arrays in `p%lidar` allocated before the assignment or are you relying on automatic allocation from Fortran 2003? @andrew-platt what do you think about allowing this? It would make life a lot easier in some cases, but I haven't seen it done in the code base.